Class/C-Array-Methods/exercise6.js

```diff
@@ -6,10 +6,21 @@ let scoresList = [45, 83, 15, 8, 62, 60, 79];
 // It should return true if the score is 60 or above
 // otherwise return false
 
+function hasPassed (score){
+    console.log(score);
+    if (score >= 60){
+        return true;
+    }else{
+        return false;
+    }
 
+}
 
 // 2. Use array.filter to create a new array with only the values in scoresList which have passed
 
+let score = scoresList.filter(hasPassed);

```

Comment:
Again I would change the name of the `score` variable here to make it clearer what it is (e.g. to `passedList`) and to make it easier to read. The code is correct though.